Quaver.Shared/Screens/Main/MainMenuScreen.cs

```diff
@@ -107,6 +114,21 @@ private void HandleKeyPressEscape()
             DialogManager.Show(new QuitDialog());
         }
 
+        private void HandleKeyPressControlF5()
```

Comment:
Not sure about doing it this way. If the user chooses to refresh here and clicks yes on the dialog, I would expect it to do it immediately rather than wait for them to go to song select.